dev-server/documents/html/whitespace.mustache

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Document with whitespace examples</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <style type="text/css">
+      p {
+        border: 1px dashed grey;
+        padding: 1em;
+      }
+
+      span {
+        border: 1px dotted pink;
+        padding: 0.5em;
+      }
+    </style>
+  </head>
+  <body>
+  <h1>HTML document with examples of whitespace</h1>
+  <p>This HTML document contains various types of whitespace, including HTML entities
+  representing Unicode whitespace characters, as well as empty elements.
+  It is intended to help test text selection.</p>
```

Comment:
Do you have a preferred reference for looking up the HTML entity codes used here? The table at https://en.wikipedia.org/wiki/Whitespace_character#Unicode seems good.